docs/modules/ROOT/pages/tutorials/install.adoc

```diff
@@ -190,3 +190,79 @@ If two instances have the same name and namespace bad things will break in unexp
 This means in practice you will want to put the production vault in a separate namespace.
 
 More information on how component instantiation works can be found https://syn.tools/commodore/reference/architecture.html#_component_instantiation[here].
+
+
+== OIDC Integration
+
+Vault has the option to authenticate users through OIDC.
+This guide describes the manual setup of the Vault OIDC integration for a Lieutenant connected Vault.
+
+[WARNING]
+====
+It should also be possible to configure OIDC integration through the component.
+However, this requires a client secret we don't recommend doing so when integrating Vault with Lieutenant as this will introduce a dependency for Vault on itself making setup and disaster recovery more difficult.
```

Comment:
```suggestion
However, this requires a client secret which should be stored in Vault.
we don't recommend configuring OIDC through the component when integrating Vault with Lieutenant as this will introduce a dependency for Vault on itself making setup and disaster recovery more difficult.
```